### ISSUE TITLE
Release tarballs should include README

### ISSUE THREAD

**Issue by annulen** [annulen/webkit#644]

**Comment by annulen** [comment]
Should be something like
```
You can build and install QtWebKit with these commands:

mkdir build
cd build
cmake -G Ninja -DPORT=Qt -DCMAKE_BUILD_TYPE=Release ..
ninja
sudo ninja install

See https://github.com/annulen/webkit/wiki for more detailed instuctions
```

**Comment by annulen** [comment]
Fixed in dc2687ac86ddff8b94fe4b383c5f423ad021f0c1
